cognite_toolkit/_cdf_tk/prototypes/_packages/update_references.py

```diff
@@ -0,0 +1,338 @@
+import inspect
```

Comment:
Need to support Python 3.9
```suggestion
from __future__ import annotations

import inspect
```